intro-to-R-tidyverse/02-intro_to_ggplot2.Rmd

```diff
@@ -259,8 +259,8 @@ ggplot(
   geom_hline(yintercept = 5.5, color = "darkgreen") # we can specify colors by names here
 ```
 
-We can change the x and y labels by using `ylab` and `xlab` functions and add a
-title using `ggtitle`.
+We can change the x and y labels, as well as add an overall plot title, using the `labs()` function.
```

Comment:
Just thinking about this a bit... Wondering if we might want to present with just the `xlab()` & `ylab()` here (no title) for the first iteration, then have the next iteration add a title and legend name while transitioning to `labs()`.  

My reasoning here is that sometimes it is nice to just have the somewhat simpler `xlab("foo")` than `labs(x = "foo")`. That is used a number of times in later examples (though maybe you got all of them), and this allows us to introduce both without much friction.